mmf/models/movie_mcan.py

```diff
@@ -24,7 +24,9 @@ def __init__(self, config):
         super().__init__(config)
         self.config = config
         self._global_config = registry.get("config")
```

Comment:
I would prefer global_config just be a variable rather than an instance variable (not owned by self). Because I really don't see it being used anywhere else. https://fburl.com/k0qak2bp

Reply:
Let's handle that in a separate PR. To unblock intern, let's first land this.